### ISSUE TITLE
[BUG]: in post composer ALT mode, alt text cannot be added for the 3rd image and beyond

### ISSUE THREAD

**Issue by suvam0451** [suvam0451/dhaaga#135]

**Comment by suvam0451** [comment]
Still exists for v0.9.0.

And yeah, it's the third image onwards.

**Comment by suvam0451** [comment]
No longer a bug, since the new quick post UI was introduced
